packages/shared/src/graphql/posts.ts

```diff
@@ -789,6 +803,25 @@ export const createPost = async (
   return res.createFreeformPost;
 };
 
+export const VOTE_POLL_MUTATION = gql`
+  mutation VotePoll($postId: ID!, $optionId: ID!, $sourceId: ID!) {
```

Comment:
Why do we need to send the `sourceId`. Is the `postId` not enough? We should be able to pull the `sourceId` when pulling the post info, right?